source/installer.py

```diff
@@ -455,7 +455,7 @@ def createPortableCopy(destPath,shouldCopyUserConfig=True):
 	tryCopyFile(os.path.join(destPath,"nvda_noUIAccess.exe"),os.path.join(destPath,"nvda.exe"))
 	if shouldCopyUserConfig:
 		copyUserConfig(os.path.join(destPath,'userConfig'))
-	removeOldLibFiles(destPath,rebootOK=False)
+	removeOldLibFiles(destPath,rebootOK=True)

```

Comment:
How about the case where a portable copy resides on a removable drive and that drive is no longer available after a reboot? I assume Windows silently ignores this?